### DISCUSSION TITLE
如果使用database-discovery为注册中心，client如何使用，有文档么

### DISCUSSION THREAD

**Discussion by liuyukuai** [apolloconfig/apollo#4858] [General]
如果使用database-discovery为注册中心，client如何使用，有文档么

**Comment by nobodyiam** [comment]
可以参考[2.2.3.5 database-discovery](https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_2235-database-discovery)

**Reply by liuyukuai** [reply]
我已经按照这个文档部署了apollo，也成功了，我启动了一个应用启动也成功了配置信息也取到了，但是该应用没注册到database-discovery中，通过8080端口访问，只有pollo-configservice和apollo-adminservice，没有我自己的应用。包括使用官方提供的spring-boot-demo的程序，也没注册上，我是漏掉什么配置了么

**Reply by liuyukuai** [reply]
如果把eureka替换成database-discovery，apollo没有服务发现的功能了么，Client也没有引用eureka组件。

**Reply by Anilople** [reply]
和通用的注册中心eureka，nacos不同，
database-discovery 设计上是仅仅提供给apollo-configservice 和 apollo-adminservice内部使用的，不对用户开放
